client/state/selectors/get-account-recovery-reset-user-first-name.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * @param {Object} state Global app state
+ * @return {String} The first name of the user making password reset request.
+ */
+export default ( state ) => {
+	return get( state, 'accountRecovery.reset.userData.firstName', '' );
```

Comment:
Should the default really be empty string? What will the UI display in that case?